common.sh

```diff
@@ -23,9 +23,8 @@ if [ "${EVENT_NAME}" == "pull_request" ]; then
   export DEBIAN_REPO_BASE_URL="https://repository.hazelcast.com/debian-test-local"
   export RPM_REPO_BASE_URL="https://repository.hazelcast.com/rpm-test-local"
 
-  # TODO change when the access is setup
-  export BREW_GIT_REPO_NAME="frant-hartm/homebrew-hz"
-  export BREW_REPO_NAME="frant-hartm/hz"
+  export BREW_GIT_REPO_NAME="hazelcast/homebrew-hz-test"
+  export BREW_REPO_NAME="hazelcast/hz-test"
```

Comment:
What do you think about deduplication with small bash magic
```suggestion
  export BREW_REPO_NAME="hazelcast/hz-test"
  export BREW_GIT_REPO_NAME="${BREW_REPO_NAME//\///homebrew-}"
```

Reply:
Also, maybe rename `BREW_REPO_NAME` to `BREW_TAP_NAME` ?